WMF Framework/EventPlatformClient/EventPlatformClient.swift

```diff
@@ -721,7 +630,10 @@ public class EPC: NSObject {
                 return
             }
             
-            guard inSample(stream: stream) else {
+            guard let config = streamConfigs[stream] else {
```

Comment:
This check is equivalent to the `guard streamConfigs.keys.contains(stream) else {` check a few lines earlier, but I like your way of writing it better. Can you delete lines 628 - 631 and add the `DDLogDebug("EPC: Event submitted to...` line inside this guard before the `return`?

Reply:
Done